jenkins/ci.suse.de/openstack-mkcloud.yaml

```diff
@@ -522,7 +522,7 @@
                   ghremote=origin
                   git config --get-all remote.${ghremote}.fetch | grep -q pull || \
                       git config --add remote.${ghremote}.fetch "+refs/pull/*/head:refs/remotes/${ghremote}/pr/*"
-                  git fetch $ghremote 2>&1 | grep -v '\[new ref\]'
+                  git fetch $ghremote 2>&1 | grep -v '\[new ref\]' || :
```

Comment:
Technically true, although it is fixing a change which was made due to log parser-related work.  I could split it but it would be a waste of time now since there is nothing else controversial in this PR.